docs/development.md

```diff
@@ -39,23 +39,31 @@ In addition, the command will also *minify* the JavaScript files to reduce the s
 
 #### Starting the dev server
 
-Run the following command:
+To start the server in the background, run the following command 
+and wait until the task exits with a `BUILD SUCCESSFUL`:
 
```

Comment:
Delete this blank line and add a blank line after the command, before `To start the server in the foreground...`. For consistency, let's follow this format:

`<previous paragraph>`
`<blank line>`
`Instructions leading up to command:`
` ```sh`
`command`
` ``` `
`<blank line>`
`<next paragraph>`